kroxylicious-systemtests/src/main/java/io/kroxylicious/systemtests/utils/KafkaUtils.java

```diff
@@ -123,7 +123,7 @@ public static boolean restartBroker(String deployNamespace, String clusterName)
             throw new KubeClusterException.NotFound("Kafka cluster name not found!");
         }
         kubeClient().getClient().pods().inNamespace(deployNamespace).withName(podName).withGracePeriod(0).delete();
-        kubeClient().getClient().pods().inNamespace(deployNamespace).withName(podName).waitUntilCondition(Objects::isNull, 60, TimeUnit.SECONDS);
+        kubeClient().getClient().pods().inNamespace(deployNamespace).withName(podName).waitUntilCondition(Objects::isNull, 5, TimeUnit.MINUTES);
```

Comment:
what's the extended wait time for?  Pulling images for the first time?

Reply:
I forgot to mention it in the PR, this timeout was causing failures when deploying it on Openshift, sometimes it takes more time to delete the kafka broker than expected because of race conditions

Reply:
Maybe not for this PR but 5 min default timeout seems very long to me! Would be nice to make it configurable.